examples/scala/src/main/scala/mytest/spark/CloudantStreaming.scala

```diff
@@ -0,0 +1,64 @@
+package mytest.spark
+
+import org.apache.spark.sql.SQLContext
+import org.apache.spark.{ SparkContext, SparkConf }
+import org.apache.spark.streaming.{ Duration, StreamingContext, Time }
+import org.apache.spark.rdd.RDD
+import com.cloudant.spark.CloudantReceiver
+
+object CloudantStreaming {
+  def main(args: Array[String]) {
+    val sparkConf = new SparkConf().setAppName("Cloudant Spark SQL External Datasource in Scala")
+
+    // Create the context with a 10 seconds batch size
+    val duration = new Duration(10000)
+    val ssc = new StreamingContext(sparkConf, duration)
+
+    val changes = ssc.receiverStream(new CloudantReceiver(Map(
+      "cloudant.host" -> "ACCOUNT.cloudant.com",
+      "cloudant.username" -> "USERNAME",
+      "cloudant.password" -> "PASSWORD",
+      "database" -> "n_airportcodemapping"), duration.milliseconds / 2))
+
```

Comment:
Why not putting this here:
```
val sc = ssc.sparkContext
val sqlContext = new SQLContext(sc)
```

What is the advantage of calling SQLContextSingleton?